tests/other.rs

```diff
@@ -91,7 +91,7 @@ fn diagnostics() {
     assert_eq!(stats.mappings, 1);
     assert_eq!(stats.despawns, 1);
     assert_eq!(stats.packets, 2);
-    assert_eq!(stats.bytes, 33);
+    assert_eq!(stats.bytes, 38);
```

Comment:
Here I changed from 33 to 38. Don't know if this is from the id's changed on the bevy size, or something on rennet.